testresults/get.go

```diff
@@ -11,26 +11,53 @@ import (
 	"github.com/pkg/errors"
 )
 
+type TestResultsGetOptions struct {
+	CedarOpts timber.GetOptions
+
+	// Request information. See cedar's REST documentation for more
+	// information:
+	// `https://github.com/evergreen-ci/cedar/wiki/Rest-V1-Usage`.
+	TaskID        string
+	DisplayTaskID string
+	TestName      string
+	Execution     int
+}
+
+func (opts *TestResultsGetOptions) validate() error {
```

Comment:
Should this use a catcher to accumulate and return all validation errors instead of just the first one?